renderlib/Origins.cpp

```diff
@@ -0,0 +1,61 @@
+#include "Origins.h"
+
+#include "AppScene.h"
+#include "Logging.h"
+#include "RenderSettings.h"
+
+void
+Origins::update(SceneView& scene)
+{
+  if (scene.scene) {
+
+    // e.g. find all selected objects in scene and collect up their centers/transforms here.
+    SceneLight& lt = scene.scene->m_lighting.m_sceneLights[1];
```

Comment:
Is this a TODO?

Reply:
yes, once the framework is completely adopted / generalized, this code won't have to access the area light in this way - it will go through something like scene.selectedObject().